WiserTaskScheduler/WiserTaskScheduler/appsettings.json

```diff
@@ -34,8 +34,7 @@
     "Name": "WTS development",
     "SecretsBaseDirectory": "C:\\WiserSecrets\\",
     "MainService": {
-      "LocalConfiguration": "C:\\WTS configurations\\WTSTestSettings.xml",
-      "LocalOAuthConfiguration": "C:\\WTS configurations\\WTSTestOAuth.xml",
+      "LocalConfiguration": "C:\\WTS configurations\\WTSTestOAuth.xml",
```

Comment:
Dit soort wijzigingen aan de appsettings niet committen aub, dit is een aanpassing waar andere mensen niets aan hebben.